ui/dashboards/src/components/Panel/Panel.tsx

```diff
@@ -100,10 +100,10 @@ export function Panel(props: PanelProps) {
           overflow: 'hidden',
           flexGrow: 1,
           margin: 0,
-          padding: (theme) => theme.spacing(panelPadding),
+          padding: 0,
```

Comment:
Looks like we might want to rename `panelPadding` variable / get rid of the comment above -- looks like it's still used for PanelHeader, mahybe we just inline it

Reply:
Good catch! I can use the value I moved up into the theme.